3.6/data-modeling-databases-working-with.md

```diff
@@ -138,8 +138,16 @@ change into the current database to the new database. Changing the current
 database must explicitly be requested by using the
 *db._useDatabase* method.
 
-The *options* attribute currently has no meaning and is reserved for
-future use.
+The *options* attribute can be used to set defaults for collections that will
+be created in the new database:
+
+* *sharding*: The sharding method to use. Valid values are: "" or "single".
+  Setting this option to "single" will enable the one-shard feature in the enterprise edition.
+* *replicationFactor*: Default replication factor. Special values include "satellite",
+  which will replicate the collection to every DB-server, and 1, which disables replication.
+* *minReplicationFactor*: Default minimum replication factor. If there are
+  less than minReplication replicas available the collection will become
+  read-only.
```

Comment:
This sounds a lot like the description of the write-concern option. Do both exist? write-concern as startup option but minReplicationFactor in the API? This question was also raised by Simon:
https://docs.google.com/document/d/1wvLKGnBPXTXvDY-N_blgyOhSudMCaUrP6sa6cHKcQ3U/edit?usp=sharing

If this means the same thing then it should have the same name, WDYT?